antelope_contracts/contracts/erc20/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+cmake_minimum_required(VERSION 3.16)
+project(erc20)
+
+find_package(cdt)
+
+set(SOURCES, "")
+
+list(APPEND SOURCES
+    ${CMAKE_CURRENT_SOURCE_DIR}/src/erc20.cpp
+)
+
+# ethash
+list(APPEND SOURCES 
+    ${EXTERNAL_DIR}/silkworm/third_party/ethash/lib/keccak/keccak.c
+    ${EXTERNAL_DIR}/silkworm/third_party/ethash/lib/ethash/ethash.cpp
+    ${EXTERNAL_DIR}/silkworm/third_party/ethash/lib/ethash/primes.c
+)
+
+# silkworm
+list(APPEND SOURCES 
+    ${EXTERNAL_DIR}/silkworm/silkworm/core/common/util.cpp
+    ${EXTERNAL_DIR}/silkworm/silkworm/core/execution/address.cpp
+    ${EXTERNAL_DIR}/silkworm/silkworm/core/common/endian.cpp
+    ${EXTERNAL_DIR}/silkworm/silkworm/core/rlp/encode.cpp
+    ${EXTERNAL_DIR}/silkworm/silkworm/core/rlp/decode.cpp
+)
+
+
+add_contract( erc20 erc20 ${SOURCES} )
+target_include_directories( erc20 PUBLIC 
+                            ${CMAKE_CURRENT_SOURCE_DIR}/include 
+                            ${CMAKE_BINARY_DIR}/../../
```

Comment:
Which header file(s) is the include directory meant for? There should be a better name used in which the necessary header files are placed to make it clear where they are from.

Better yet, you should use the target if possible to automatically get the public include paths of the target to use here.

Reply:
fixed